src/synapse-plugins/converter-plugin/converter-data.vala

```diff
@@ -0,0 +1,214 @@
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uscup", "cup", NC_(VOLUME, "US cup"), "8", "usfluidounce"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uslegalcup", "legalcup|cup", NC_(VOLUME, "US legal cup"), "240/1000", "liter"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "metriccup", "cup", NC_(VOLUME, "US metric cup"), "250/1000", "liter"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "ustablespoon", "ustbsp|tbsp", NC_(VOLUME, "US tablespoon"), "1/16", "uscup"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "uslegaltablespoon", "legaltbsp|tbsp", NC_(VOLUME, "US legal tablespoon"), "1/16", "uslegalcup"},
+        {UnitType.DIMENSION, UnitSystem.IMPERIAL_US, "usteaspoon", "ustsp|tsp", NC_(VOLUME, "US teaspoon"), "1/3", "ustablespoon"},
+
+
+        //Time units
+        {UnitType.TIME, UnitSystem.SI, "second", "sec|s", NC_(TIME, "second"), "1", ""}, // Fundamental
+        {UnitType.TIME, UnitSystem.SI, "minute", "min|m", NC_(TIME, "minute"), "60", "second"},
+        {UnitType.TIME, UnitSystem.SI, "hour", "hr|h", NC_(TIME, "hour"), "60", "minute"},
+        {UnitType.TIME, UnitSystem.SI, "day", "da|d", NC_(TIME, "day"), "24", "hour"},
+        {UnitType.TIME, UnitSystem.SI, "week", "wk", NC_(TIME, "week"), "7", "day"},
+        {UnitType.TIME, UnitSystem.SI, "fortnight", "", NC_(TIME, "fortnight"), "14", "day"},
+        {UnitType.TIME, UnitSystem.SI, "commonyear", "yr|y|calendaryear", NC_(TIME, "Common year"), "365", "day"},
+        {UnitType.TIME, UnitSystem.SI, "leapyear", "yr|y", NC_(TIME, "leap year"), "366", "day"},
+        {UnitType.TIME, UnitSystem.SI, "julianyear", "yr|y", NC_(TIME, "Julian year"), "365.25", "day"},
+        {UnitType.TIME, UnitSystem.SI, "gregorianyear", "yr|y", NC_(TIME, "Gregorian year"), "366.2425", "day"},
+        {UnitType.TIME, UnitSystem.SI, "islamicyear", "yr|y", NC_(TIME, "Islamic year"), "354", "day"},
+        {UnitType.TIME, UnitSystem.SI, "islamicleapyear", "yr|y", NC_(TIME, "Islamic leap year"), "354", "day"},
+        {UnitType.TIME, UnitSystem.SI, "decade", "", NC_(TIME, "decade"), "10", "year"},
+        {UnitType.TIME, UnitSystem.SI, "century", "", NC_(TIME, "century"), "100", "year"},
+        {UnitType.TIME, UnitSystem.SI, "millenium", "", NC_(TIME, "millenium"), "1000", "commonyear"},
+
+        // Velocity units  - at present treated as separate unit system although could be calculated from components
+        {UnitType.VELOCITY, UnitSystem.SI, "meterpersecond", "m/s", NC_(TIME, "meters per second"), "1", ""}, // Fundamental
+        {UnitType.VELOCITY, UnitSystem.SI, "lightspeed", "c", NC_(TIME, "speed of light"), "299792458", "meterpersecond"},
+        {UnitType.VELOCITY, UnitSystem.SI, "kilometersperhour", "kph", NC_(TIME, "kilometers per hour"), "1000/3600", "meterpersecond"},
+        {UnitType.VELOCITY, UnitSystem.SI, "milesperhour", "mph", NC_(TIME, "miles per hour"), "1609.34/3600", "meterpersecond"},
```

Comment:
```suggestion
        {UnitType.VELOCITY, UnitSystem.SI, "milesperhour", "m/h|mph", NC_(TIME, "miles per hour"), "1609.34/3600", "meterpersecond"},
```

Reply:
I've not seen "m/h" used as an abbreviation but it does no harm to add it I guess (it is unlikely to be meters per hour)